diff --git a/lib-es/Transport.d.ts b/lib-es/Transport.d.ts
index e2f9326dc33382a43b0128d6ec443fd15254b3c0..8089597390412008db8dc546c76d76d4b03bd4f7 100644
--- a/lib-es/Transport.d.ts
+++ b/lib-es/Transport.d.ts
@@ -130,7 +130,7 @@ export default class Transport {
      * @returns {Promise<void>} A promise that resolves when the transport is closed.
      */
     close(): Promise<void>;
-    _events: EventEmitter<[never]>;
+    _events: EventEmitter;
     /**
      * Listen for an event on the transport instance.
      * Transport implementations may have specific events. Common events include:
diff --git a/lib/Transport.d.ts b/lib/Transport.d.ts
index e2f9326dc33382a43b0128d6ec443fd15254b3c0..8089597390412008db8dc546c76d76d4b03bd4f7 100644
--- a/lib/Transport.d.ts
+++ b/lib/Transport.d.ts
@@ -130,7 +130,7 @@ export default class Transport {
      * @returns {Promise<void>} A promise that resolves when the transport is closed.
      */
     close(): Promise<void>;
-    _events: EventEmitter<[never]>;
+    _events: EventEmitter;
     /**
      * Listen for an event on the transport instance.
      * Transport implementations may have specific events. Common events include:
