diff --git a/scripts/cocoapods/new_architecture.rb b/scripts/cocoapods/new_architecture.rb
index 7d8f8e64fc0f53a730da8d3ffa4a7c1f85a53f28..aa26f6200e896d6c8b4453f4a7bab6939c74c5ef 100644
--- a/scripts/cocoapods/new_architecture.rb
+++ b/scripts/cocoapods/new_architecture.rb
@@ -180,7 +180,12 @@ class NewArchitectureHelper
                 next if should_skip
 
                 # Read the file as a plist
-                info_plist = Xcodeproj::Plist.read_from_path(infoPlistFile)
+                begin
+                    info_plist = Xcodeproj::Plist.read_from_path(infoPlistFile)
+                rescue StandardError => e
+                    Pod::UI.warn("Failed to read Info.plist at #{infoPlistFile}: #{e.message}")
+                    next
+                end 
                 # Check if it contains the RCTNewArchEnabled key
                 if info_plist["RCTNewArchEnabled"] and info_plist["RCTNewArchEnabled"] == new_arch_enabled
                     next
